src/Command/Command/SimpleTrader/Monitor.php

```diff
@@ -41,11 +41,16 @@ protected function configure()
         $this->setDescription('Monitors the status of placed orders for the simple trade repeater');
     }
 
-    protected function initialize($input, $output)
+    /**
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @throws \Kobens\Core\Exception\LogicException
+     */
+    protected function initialize($input, $output): void
     {
         $this->repeater = new SimpleRepeater();
-        $this->mapper = new Mapper();
-        $this->log = new Logger('simple_trade_monitor');
+        $this->mapper   = new Mapper();
```

Comment:
This mapper class is very much a utility function. I'm debating turning it into a single static method for accessing mapped values.